andela/travel_tool_front
# #160718924 Add Mobile Responsiveness to the New Request Form

#### What does this PR do?
- Adds mobile responsiveness to the New Request Form

#### Description of Task to be completed?
- Ensure that the form appears well on mobile device screens

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout bg-request-form-responsiveness-160718924`
- Install dependencies - `yarn install`
- Start the server  - `make start`
- Navigate to `http://local.andela.com:3000/requests` 
- Login if you're not logged in
- Click on the `New Request` Button
- Use your browsers developer tools to simulate the appearance of the form on difference screen sizes

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
 #160718924

#### Screenshots (if appropriate)
###### On iPad screen size

<img width="578" alt="screen shot 2018-10-15 at 08 34 30" src="https://user-images.githubusercontent.com/23254512/46931767-35e81580-d055-11e8-91fa-47a935f8ee18.png">

###### On iPhone screen size
<img width="306" alt="screen shot 2018-10-15 at 08 33 44" src="https://user-images.githubusercontent.com/23254512/46931806-6334c380-d055-11e8-90b7-c2848f43ea97.png">

#### Questions:
